### DISCUSSION TITLE
报错 No module named 'fsspec.asyn' 咋解决啊

### DISCUSSION THREAD

**Discussion by liyi-la** [GaiZhenbiao/ChuanhuChatGPT#350] [Q&A]
运行python ChuanhuChatbot.py后报错 No module named 'fsspec.asyn'，这个好像是gradio里面的东西，但是我已经安装了gradio库，咋解决啊

**Comment by Flesy-M** [comment]
fsspec卸了重装一遍

**Reply by enihs** [reply]
重装没用咋办啊

**Comment by enihs** [comment]
ModuleNotFoundError: No module named 'gradio' 我一直是这个错误，重装几遍 都没用

**Reply by Flesy-M** [reply]
gradio下还有好多包，可以具体看看哪些报错，把特定的包更新一下或者卸载重装
